src/EPR.PRN.Backend.Obligation.UnitTests/Services/ObligationCalculatorServiceTests.cs

```diff
@@ -6,26 +6,22 @@
 using FluentAssertions;
 using Microsoft.Extensions.Logging;
 using Moq;
+using System.Diagnostics.CodeAnalysis;
 
 namespace EPR.PRN.Backend.Obligation.UnitTests.Services;
 
+[ExcludeFromCodeCoverage]
```

Comment:
remove

Reply:
We don't need to look for code coverage on unit tests